docs/workloads/realtime/configuration.md

```diff
@@ -66,14 +66,13 @@
   predictor:
     type: tensorflow
     path: <string>  # path to a python file with a TensorFlowPredictor class definition, relative to the Cortex root (required)
-    model_path: <string>  # S3 path to an exported model directory (e.g. s3://my-bucket/exported_model/) (either this or 'models' must be provided)
-    signature_key: <string>  # name of the signature def to use for prediction (required if your model has more than one signature def)
-    models:  # use this to serve multiple models in a single API (either this or 'model_path' must be provided)
-      dir: <string>  # S3 path to a directory containing multiple models (e.g. s3://my-bucket/models/) (either this or 'paths' must be provided)
-      paths:  # list of S3 paths to exported model directories (either this or 'dir' must be provided)
+    models:  # use this to serve a single model or multiple ones
+      dir: <string>  # S3 path to a directory containing multiple models (e.g. s3://my-bucket/models/)
+      path: <string> # S3 path to an exported model directory (e.g. s3://my-bucket/exported_model/)
+      paths:  # list of S3 paths to exported model directories
         - name: <string>  # unique name for the model (e.g. text-generator) (required)
-          model_path: <string>  # S3 path to an exported model directory (e.g. s3://my-bucket/exported_model/) (required)
-          signature_key: <string>  # name of the signature def to use for prediction (required if your model has more than one signature def)
+          path: <string>  # S3 path to an exported model directory (e.g. s3://my-bucket/exported_model/) (required)
+          signature_key: <string>  # name of the signature def to use for prediction (optional)
```

Comment:
nit: `name of the signature def to use for prediction (required if your model has more than one signature def)`